sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -12,6 +12,7 @@ import { BatchingReceiver } from "./batchingReceiver";
 import { AbortSignalLike } from "@azure/abort-controller";
 import { throwErrorIfConnectionClosed } from "./util/error";
 import { EventPosition } from "./eventPosition";
+import "@azure/core-asynciterator-polyfill";
```

Comment:
If we are not using `@azure/core-asynciterator-polyfill` in `receiver.ts`, then why are we importing here?

Reply:
This is a pollyfill, so when you import the library, it's actually checking if the symbol `asyncIterator` exists in the runtime environment, and if it doesn't it creates it.